# awesome-proteomics

### Introduction 
* Youtube playlist by Nikolai Slavov: [Statistical significance of differential protein abundance | Statistics for proteomics](https://www.youtube.com/watch?v=teG856EiZAg&list=PLHLRxq8iKFsKrwd6QeKh2pQh6Hzj7V-jT&index=4) It has a couple of videos to get you started on proteomics data analysis. He is a good follow on Twitter/X.
 
* Laurent Gatto works a lot on proteomics https://lgatto.github.io/. I had the pleasure meeting him years ago when he visited Boston. 

* [Galaxy proteomics training](https://training.galaxyproject.org/training-material/topics/proteomics/)

* [Bioinformatics for Proteomics](https://www.compomics.com/bioinformatics-for-proteomics/) The CompOmics group, headed by Prof. Dr. Lennart Martens, is part of the Department of Biomolecular Medicine of the Faculty of Medicine and Health Sciences of Ghent University,

### pipelines and statistical tools 

* [DEqMS](https://www.bioconductor.org/packages/release/bioc/html/DEqMS.html)
  
* [Embrace R to Boost Your Proteomic Analysis](https://towardsdatascience.com/embrace-r-to-boost-your-proteomic-analysis-ea6fdc8909e7)

* [Autoprot](https://www.biorxiv.org/content/10.1101/2024.01.18.571429v1): Processing, Analysis and Visualization of Proteomics Data in Python 

* [Initial recommendations for performing, benchmarking and reporting single-cell proteomics experiments | Nature Methods](https://www.nature.com/articles/s41592-023-01785-3)

* [scp](https://github.com/UCLouvain-CBIO/scp) package: Single cell proteomics data processing 

* [quantms: nextflow pipeline for proteomics](https://nf-co.re/quantms/1.1.1)

* The goal of [protti](https://jpquast.github.io/protti/) is to provide flexible functions and workflows for proteomics quality control and data analysis, within a single, user-friendly R package 

* [Sage: proteomics searching so fast it seems like magic](https://github.com/lazear/sage)

* [Protocol for the processing and downstream analysis of phosphoproteomic data with PhosR](https://star-protocols.cell.com/protocols/701)
  
* [Fragpipe](https://fragpipe.nesvilab.org/) A complete proteomics pipeline with the MSFragger search engine at heart

* [MSstats Version 4.0](https://pubs.acs.org/doi/10.1021/acs.jproteome.2c00834): Statistical Analyses of Quantitative Mass Spectrometry-Based Proteomic Experiments with Chromatography-Based Quantification at Scale 

* [MSstatsTMT](https://vitek-lab.github.io/MSstatsTMT/articles/MSstatsTMT.html) : A package for protein significance analysis in shotgun mass spectrometry-based proteomic experiments with tandem mass tag (TMT) labeling

* [MS-DAP Platform](https://pubs.acs.org/doi/10.1021/acs.jproteome.2c00513) for Downstream Data Analysis of Label-Free Proteomics Uncovers Optimal Workflows in Benchmark Data Sets and Increased Sensitivity in Analysis of Alzheimerâ€™s Biomarker Data 

### Databases

1/ [CPTAC](https://proteomic.datacommons.cancer.gov/pdc/), the biggest database for cancer python package to access it https://github.com/PayneLab/cptac

2/ PRIDE (PRoteomics IDEntifications) https://www.ebi.ac.uk/pride/ is a rich resource of mass spectrometry based proteomics data. It has cancer data and more. 

3/ And there are some handy websites to conveniently check proteins you have in mind such as Uniprot (https://www.uniprot.org/), Nextprot (https://www.nextprot.org/), the human protein atlas (https://www.proteinatlas.org/) and PhosphoSitePlus (https://buff.ly/3ILhSLK). 

4/ two papers https://pubmed.ncbi.nlm.nih.gov/33657365/ and https://pubmed.ncbi.nlm.nih.gov/31495571/

5/ GTEx proteomics data for a variety of human tissues (https://pubmed.ncbi.nlm.nih.gov/32916130/) 

6/ CCLE cell lines have both RNA and proteomics data https://depmap.org/portal/data_page/?tab=customDownloads
